docs/user-manual.md

```diff
@@ -30,6 +30,17 @@ To stop the pie, follow these steps:
    $ CRONJOBS_LIST=$(kubectl -n ${NAMESPACE} get cronjobs --no-headers=true | awk '{print $1}')
    $ [ -n "${CRONJOBS_LIST}" ] && kubectl -n ${NAMESPACE} delete cronjobs ${CRONJOBS_LIST}
```

Comment:
Would you check Would you check is `--wait=false` option required to avoid waiting until deleting pods?

Reply:
When I tested it in my local environment, `--wait=false` wasn't necessary. It may take a while if there are many cronjobs, but it will work without this option.
```
$ kubectl get pod -n e2e
NAME                  READY   STATUS    RESTARTS   AGE
pie-88cc77c4c-kp9xb   1/1     Running   0          18s

$ kubectl get cronjob -ne2e
NAME                                        SCHEDULE         SUSPEND   ACTIVE   LAST SCHEDULE   AGE
pie-probe-pie-test-contro-standard-d71204   0-59/1 * * * *   False     0        <none>          4s

$ kubectl -n e2e scale deploy --replicas=0 pie
deployment.apps/pie scaled

$ kubectl get pod -n e2e
NAME                                                       READY   STATUS   RESTARTS   AGE
pie-probe-pie-test-contro-standard-d71204-28577272-655zj   0/1     Error    0          32s
pie-probe-pie-test-contro-standard-d71204-28577272-cnpz7   0/1     Error    0          92s
pie-probe-pie-test-contro-standard-d71204-28577272-fz5f9   0/1     Error    0          67s

$ kubectl delete cronjob -n e2e pie-probe-pie-test-contro-standard-d71204
cronjob.batch "pie-probe-pie-test-contro-standard-d71204" deleted
### ^^^^^ Successfully deleted and exited normally

❯ kubectl get cronjob -n e2e
No resources found in e2e namespace.

```